common/fireblocks_config.go

```diff
@@ -0,0 +1,57 @@
+package common
+
+import (
+	"github.com/urfave/cli"
+)
+
+const (
+	FireblocksAPIKeyFlagName           = "fireblocks-api-key"
+	FireblocksAPISecretPathFlagName    = "fireblocks-api-secret-path"
+	FireblocksBaseURLFlagName          = "fireblocks-api-url"
+	FireblocksVaultAccountNameFlagName = "fireblocks-vault-account-name"
+)
+
+type FireblocksConfig struct {
+	APIKey           string
+	SecretKeyPath    string
+	BaseURL          string
+	VaultAccountName string
+}
+
+func FireblocksCLIFlags(envPrefix string, flagPrefix string) []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
+			Name:     PrefixFlag(flagPrefix, FireblocksAPIKeyFlagName),
+			Usage:    "Fireblocks API Key. To configure Fireblocks MPC wallet, this field is required.",
```

Comment:
comment that they can be empty string, in which case it'll fallback to private key wallet

Reply:
It's an optional parameter, so empty string doesn't need to be passed in to fall back to private key wallet.
I'll comment that it will fall back to private key wallet if not configured.